perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -22,6 +22,25 @@
 
 As other cloud providers deliver a managed MySQL service, we will add it here.
 
+To make this benchmark run for GCP:
+To run this benchmark for GCP if is required to install a non-default gcloud
```

Comment:
I see the reasons are specified below, properly better to pull that up to here for clarity.